content/faq/41_what-if-i-could-not-find-anyone-for-review/index.md

```diff
@@ -0,0 +1,15 @@
+---
+title: what if I couldn't find anyone around for review/recommendation? Am I going to be eligible?
+type: "faq"
+priority: "41"
+tags: ["reviews", "recommendations", "3R"]
```

Comment:
2R

Reply:
I just noticed this. I actually copied this from qna no. 6. Should i also change 3R to 2R there?

Reply:
yes. please change this there as we ll.